README.md

```diff
@@ -16,14 +16,14 @@ A read-only preview with mock data is available at http://konveyor-virt-ui-previ
 Clone and install dependencies:
 
 ```bash
-git clone https://github.com/konveyor/virt-ui
-cd virt-ui
+git clone https://github.com/konveyor/forklift-ui
+cd forklift-ui
 yarn install
 ```
 
-Create a virtMeta.config.json file in the config directory using [`config/virtMeta.dev.example.json`](https://github.com/konveyor/virt-ui/blob/master/config/virtMeta.example.json) as a template. Set the `inventoryApi` property to the root URL of your virt-controller inventory API, and set the `clusterApi` property to the root URL of your host OpenShift cluster API. And also to be able to use VMware provider data to be analysed by Migration Analytics set the `inventoryPayloadApi` property to the root URL of your virt-controller inventory Payload API.
+Create a virtMeta.config.json file in the config directory using [`config/virtMeta.dev.example.json`](https://github.com/konveyor/forklift-ui/blob/master/config/virtMeta.example.json) as a template. Set the `inventoryApi` property to the root URL of your virt-controller inventory API, and set the `clusterApi` property to the root URL of your host OpenShift cluster API. And also to be able to use VMware provider data to be analysed by Migration Analytics set the `inventoryPayloadApi` property to the root URL of your virt-controller inventory Payload API.
```

Comment:
virt-controller is referenced here, is that also changing to forklift-controller?

Reply:
Yes indeed.